ktoml-core/src/jvmMain/kotlin/com/akuleshov7/ktoml/utils/RegexesJvm.kt

```diff
@@ -0,0 +1,25 @@
+/**
+ * Regexes used in tree writing.
+ */
+
+package com.akuleshov7.ktoml.utils
+
+import kotlin.text.RegexOption.COMMENTS
+
+internal actual val controlCharacterRegex = Regex("""[\p{Cc}\x7F&&[^\t]]""")
```

Comment:
could you please add comment of what kind of regex you have here and why

Reply:
`&&` is a class intersection and `\p{Cc}` is a unicode category, both only present on the JVM. The less compact version for all other platforms is clearer though, should we use that instead and avoid multiple variations? I'll document it for now

Reply:
Actually, I'll just remove it. Adds unnecessary complexity to the project